### PR TITLE
Implementing really simple Tag classes

### LINKED ISSUE #155: Single type argument version of Tag
The current API provides two versions of `Tag`.
* `Tag`
* `Tag<M,T>`

This issue suggests that we add a third type that takes a guess at which Mapper class to use based on a single type argument for the CLR type to use. i.e.
`Tag<T>`

Some discussions where this was discussed:
* https://github.com/libplctag/libplctag.NET/issues/28#issuecomment-653854167
* https://github.com/libplctag/libplctag.NET/issues/153

What needs to be decided?
* What should the mappings be between CLR type and Mapper types.

### PR DESCRIPTION
**Pull request by jkoplo**
Closes #155 

I felt a little dirty writing this code, but the intended end user is for devs that are new-er to C#. Generics can be confusing and even more so when a class is defined with two generics. This is really just some helper syntax to keep things simpler. I put these definitions in a single file in their own `libplctag.Datatypes.Simple` - I figured this should help keep them out of the way of more experienced developers.

I'm open to using different naming conventions for the namespace and for the classes - these were just the first that came to mind. Also, if we do want to merge this, I'll update our main page README to highlight the new simple classes.

### CODE REVIEW

**Comment by jkoplo** [comment]
@timyhac how do you feel about this?

**Review by timyhac — APPROVED** [review]
İ know what you mean @jkoplo İt doesn't feel right leaving the generics behind, but agree that it makes sense in this case.
All good 

İ dunno why i didn't see these comments earlier in the week.. sorry about that.


I'm still yet to implement the base class+concrete class idea but there could be some overlap

**Comment by jkoplo** [comment]
I merged #217 into this branch and then updated the README with a simple example. If you still approve, please merge this after #217.

**Review by timyhac — APPROVED** [review]

**Comment by timyhac** [comment]
@jkoplo - are you able to resolve those documentation changes on #217 and then we can merge this PR.

**Comment by jkoplo** [comment]
#217 should be good to go. This one is ready as soon as we merge #217.

**Pull request merged**
